install_environment.sh

```diff
@@ -4,6 +4,9 @@ set -ex
 
 apt-get update
 
+# get arguments as an array
+args=("$@")
+
 # Determine external IP address
 set +x
 IP=`ifconfig eth1 | grep inet | awk '{print $2}' | sed 's/addr://'`
```

Comment:
Please pass `IP` and `hostname` via arguments as done for 'magento_dir' now and remove lines from 10 to 24 (which allow automatically determine these values)